challenges-services/src/main/java/org/exoplatform/challenges/rest/ChallengeRest.java

```diff
@@ -81,7 +88,8 @@ public Response getChallengeById(@ApiParam(value = "Challenge technical id", req
     String currentUserId = Utils.getCurrentUser();
     try {
       Challenge challenge = challengeService.getChallengeById(challengeId, currentUserId);
-      return Response.ok(EntityMapper.fromChallenge(challenge)).build();
+      List<Announcement>  announcementList = announcementService.findAllAnnouncementByChallenge(challengeId,0,0);
```

Comment:
Could you add the offset and limit as parameters with default values if they arfe missing